client/tests/integration/components/project-geography-test.js

```diff
@@ -0,0 +1,210 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import {
+  render,
+  click,
+  fillIn,
+  triggerKeyEvent,
+} from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Integration | Component | project-geography', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('user can search and add a new bbl', async function(assert) {
+    // array of bbl objects is dcp_dcp_projectbbl_dcp_pasform
+    this.server.create('bbl', {
+      dcpBblnumber: '3071590111',
+      dcpDevelopmentsite: null,
+      dcpPartiallot: null,
+    });
+
+    this.server.create('bbl', {
+      dcpBblnumber: '3071590115',
+      dcpDevelopmentsite: null,
+      dcpPartiallot: null,
+    });
+
+    const bbl1Model = await this.owner.lookup('service:store').findRecord('bbl', 1);
+    const bbl2Model = await this.owner.lookup('service:store').findRecord('bbl', 2);
+
+    const arrayOfBbls = [bbl1Model, bbl2Model];
+
+    this.bbls = arrayOfBbls;
```

Comment:
Such a teensy nit, but we know it's an array of bbls just from teh syntax aloe :) you could even do this:

```
this.bbls = [
  await this.owner.lookup('service:store').findRecord('bbl', 1),
  await this.owner.lookup('service:store').findRecord('bbl', 1),
];
```

In a lot of places, you actually name the objects themselves very well, concise, no need for redundant.

Reply:
sounds good